otlp/logs_test.go

```diff
@@ -362,7 +394,15 @@ func TestCanExtractBody(t *testing.T) {
 					{Key: "key3", Value: &common.AnyValue{Value: &common.AnyValue_BoolValue{BoolValue: true}}},
 				},
 			}}},
-			expectedValue: "{\"key1\":\"value1\",\"key2\":2,\"key3\":true}\n",
+			expectedValue: map[string]interface{}{
+				"body.key1":        "value1",
+				"body.key2":        int64(2),
+				"body.key3":        true,
```

Comment:
For kvlists, I wonder if we should drop the `body` prefix.

Pros:

- Less prefixes which is nice maybe
- When using something like fluentbit, all attributes I add end up in the body even when not wanted (it's a bug they won't fix), and this would solve for that

Cons:

- Could have to de-dupe fields
- No way to distinguish between an attribute you added outside the log body and an attribute that lives within the log body (but does that even matter)

For arrays I think we would not do this though, since "0" and "1" are bad key names.

Reply:
Note: I don't have a strong preference either way, but it'll be a decision we take intentionally since it'll be breaking if we change back.

Reply:
I'll resolve this. Talked about it and it seems like there's consensus to keep the `body.` prefix.